bakery

```diff
@@ -73,6 +73,11 @@ if [[ ! x${11} == "x" ]]; then
   log info "using bake_volume_size ${11}"
 fi
 
+if [[ ! x${12} == "x" ]]; then
```

Comment:
maybe we should just make this an environment variable override instead yet another cli arg until we refactor this to name params